device.go

```diff
@@ -31,6 +31,11 @@ type Device interface {
 	IsBrowser() bool
 	HasKeyboard() bool
 	SystemScaleForWindow(Window) float32
+
+	// SetDisableScreenBlanking allows an app to ask the device not to sleep/lock/blank displays
```

Comment:
The `Device` interface comment will need updating since it no longer just "provides information" about the device

Reply:
Alternatively, the API could be on the Driver type

Reply:
Good point - best not to change the intent of an interface ;)